src/main/webapp/partials/instructorHelpSessions.jsp

```diff
@@ -2012,9 +2012,37 @@
       <br>
       <br>
     </li>
+    <li id="fbSessionResendLinksRequest">
+      <span class="text-bold">
+          <b>5. Requesting resending of access links</b>
+      </span>
+      <p>
+        TEAMMATES allows students to request for resending of access links which students might accidentally deleted. Students can do so by themselves without the need to approach the instructor.
```

Comment:
"... request for access links which students might accidentally deleted to be resent ..."